README.md

```diff
@@ -0,0 +1,24 @@
+
+# Notebook Images
+
+These images were created to be used with Open Data Hub (ODH) with the ODH Notebook Controller as the launcher.
+
+## Container Image Layering
+
+```mermaid
+
+flowchart TD
+
+ubi8/python-3.8 --> Base(base-ubi8-py38);
+
+```
+
+### Base Notebook Image
+
+We use the Red Hat UBI image `ubi8/python-38` [registry.access.redhat.com/ubi8/python-38](https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a) as builder image for the notebook base.
```

Comment:
For the ubi8 link we can use:
- [ubi8/python-38](https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a) or
- [registry.access.redhat.com/ubi8/python-38](https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a) (recommended)

But using both (`ubi8/python-38` [registry.access.redhat.com/ubi8/python-38]) looks ugly.